torchbox/torchbox.com
# Enforce global date formatting

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/views/6453316)

### Description of Changes Made
- Format all dates on the site from one centralised file which can easily be (re)configured as required
- The dates have all be changed to match the format Y-m-d which would look like 1 Feb 2024

**Post-deployment:**
When this goes live on production, we should ask the SEO team to double-check that the altered format of the meta_tag published field hasn't had any unintended consequences. See [note on ticket](https://torchbox.monday.com/boards/1192293412/pulses/1375206396/posts/167701103).

### How to Test
- Visit either a blog, work or historical work page to see the date it was published (see screenshot below)

### Screenshots
![image](https://github.com/torchbox/torchbox.com/assets/68611528/a71f1320-74bf-41d9-8e6d-ac703a2c1e41)

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
